template.go

```diff
@@ -58,6 +58,9 @@ const TEMPLATE = `
 				{{range $index, $cronv := .CronEntries}}
 					{{ $job := JSEscapeString $cronv.Crontab.Job }}
 					tasks['{{$job}}'] = [];
```

Comment:
The following solution is simpler:

```suggestion
					tasks['{{$job}}'] = tasks['{{$job}}'] || [];
```

Reply:
excellent👍
I learned a lot today. Thanks!
I think this is fine.